contracts/cairo/deploy.sh

```diff
@@ -6,37 +6,41 @@ PURPLE='\033[1;34m'
 PINK='\033[1;35m'
 COLOR_RESET='\033[0m'
 
-HERODOTUS_FACTS_REGISTRY=0x01b2111317EB693c3EE46633edd45A4876db14A3a53ACDBf4E5166976d8e869d
-MM_ETHEREUM_WALLET=0xE8504996d2e25735FA88B3A0a03B4ceD2d3086CC
-NATIVE_TOKEN_ETH_STARKNET=0x049d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7
-
 cd "$(dirname "$0")"
 
 load_env() {
     unamestr=$(uname)
     if [ "$unamestr" = 'Linux' ]; then
-      export $(grep -v '^#' ../../mm-bot/.env | xargs -d '\n')
+      export $(sed '/^#/d; s/#.*$//' .env | xargs -d '\n')
     elif [ "$unamestr" = 'FreeBSD' ] || [ "$unamestr" = 'Darwin' ]; then
-      export $(grep -v '^#' ../../mm-bot/.env | xargs -0)
+      export $(sed '/^#/d; s/#.*$//' .env | xargs -0)
     fi
 }
 load_env
 
 echo -e "${GREEN}\n=> [SN] Declare Escrow${COLOR_RESET}"
-ESCROW_CLASS_HASH=$(starkli declare --watch --rpc $SN_RPC_URL target/dev/yab_Escrow.contract_class.json)
+ESCROW_CLASS_HASH=$(starkli declare \
+  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE \
+  --watch target/dev/yab_Escrow.contract_class.json)
 
 echo -e "- ${PURPLE}[SN] Escrow ClassHash: $ESCROW_CLASS_HASH${COLOR_RESET}"
 echo -e "- ${PURPLE}[SN] Herodotus Facts Registry: $HERODOTUS_FACTS_REGISTRY${COLOR_RESET}"
-echo -e "- ${PURPLE}[SN] Market Maker: $SN_WALLET_ADDR${COLOR_RESET}"
+echo -e "- ${PURPLE}[SN] Market Maker: $MM_SN_WALLET_ADDR${COLOR_RESET}"
 echo -e "- ${PURPLE}[SN] Ethereum ContractAddress $NATIVE_TOKEN_ETH_STARKNET${COLOR_RESET}"
 echo -e "- ${PINK}[ETH] Ethereum ContractAddress: $ETH_CONTRACT_ADDR${COLOR_RESET}"
 echo -e "- ${PINK}[ETH] Market Maker: $MM_ETHEREUM_WALLET${COLOR_RESET}"
 
 echo -e "${GREEN}\n=> [SN] Deploy Escrow${COLOR_RESET}"
-ESCROW_CONTRACT_ADDRESS=$(starkli deploy --watch --rpc $SN_RPC_URL $ESCROW_CLASS_HASH \
+ESCROW_CONTRACT_ADDRESS=$(starkli deploy \
+  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE \
+  --watch $ESCROW_CLASS_HASH \
     $HERODOTUS_FACTS_REGISTRY \
     $ETH_CONTRACT_ADDR \
     $MM_ETHEREUM_WALLET \
-    $SN_WALLET_ADDR \
+    $MM_SN_WALLET_ADDR \
     $NATIVE_TOKEN_ETH_STARKNET)
 echo -e "- ${PURPLE}[SN] Escrow ContractAddress: $ESCROW_CONTRACT_ADDRESS${COLOR_RESET}"
+
+cd ../..
+contracts/solidity/set_escrow.sh $ESCROW_CONTRACT_ADDRESS
+contracts/solidity/set_withdraw_selector.sh $ESCROW_WITHDRAW_SELECTOR
```

Comment:
This script should only do the deployment to starknet. If we want to do it with just one command the deployment + finalize the setup of ethereum we should have that connection made with the make target.

I suggest removing this from here and adding this functionality to a new makefile target called `sn_deploy_and_connect_eth` or something in those lines . 

This are the steps this command will execute:
1. deploy starknet contracts (calling `deploy.sh` in the cairo folder)
2. execute set_escrow.sh with the new escrow address from starknet
3. execute  set_withdraw_selector.sh with the new escrow withdraw selector

Reply:
Also, make sure to explain how to do both ways. 
Explain that there are two ways before starting with anyone, so the user can choose correctly what they want to do